### PR TITLE
Updating Signi.com connector with regards to ISV comment, attempt #2

### PR DESCRIPTION
**Pull request by dk-klein**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation - no, this is a connector fix
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

This fix is made to address an issue that was brought up via the Activity Control pane in ISV studio. The message I have gotten was: 
_We have found build issues coming due to the type of hidden property in your submission. It should be of type string and under double quotes. Could you please make the necessary changes and update the hidden property as string with double quotes everywhere and resubmit. For ex: "hidden": true, should be replaced by "hidden": "true"_

This is attempt #2 at covering this, as I was informed that there were issues with the Api Properties validator and that I should abandon and reopen the PR.

### CODE REVIEW

File: certified-connectors/Signi/apiProperties.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
           "description": "Client (or Application) ID of the Azure Active Directory application.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"
```
Comment (vmanoharas) [inline certified-connectors/Signi/apiProperties.json:38]: Please check apiproperties validation errors and change the value of hidden parameter duly.

File: certified-connectors/Signi/apiProperties.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
           "description": "Client secret of the Azure Active Directory application.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"
```
Comment (vmanoharas) [inline certified-connectors/Signi/apiProperties.json:49]: Please check apiproperties validation errors and change the value of hidden parameter duly.

File: certified-connectors/Signi/apiProperties.json [hunk 3]
```diff
@@ -69,7 +69,7 @@
           "description": "The resource you are requesting authorization to use.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"
```
Comment (vmanoharas) [inline certified-connectors/Signi/apiProperties.json:72]: Please check apiproperties validation errors and change the value of hidden parameter duly.

File: certified-connectors/Signi/apiProperties.json [hunk 4]
```diff
@@ -88,7 +88,7 @@
           "description": "Grant type",
           "constraints": {
             "required": "false",
-            "hidden": true,
+            "hidden": "true",
```
Comment (vmanoharas) [inline certified-connectors/Signi/apiProperties.json:91]: Please check apiproperties validation errors and change the value of hidden parameter duly.

**Review by vmanoharas — COMMENTED** [review]
Please resolve review comments.

**Review by vmanoharas — COMMENTED** [review]
Hi @dk-klein, 
Can you please fix validation errors and resolve review comments.

**Comment by dk-klein** [comment]
Hello, this PR was opened based on feedback received in the Activity Control section of ISV Studio. However, it turned out that the feedback was incorrect, so we have proceeded with the certification process using a different PR that has already been merged. Therefore, I am closing this PR. Thanks.

**Comment by vmanoharas** [comment]
Hello @dk-klein, 

Thank you very much for the clarification.

**Pull request closed without merging**